conf/deprecations.json

```diff
@@ -0,0 +1,10 @@
+{
+    "rules": {
+        "no-comma-dangle": {
+            "replacedBy": ["comma-dangle"]
```

Comment:
Originally I was trying to keep it open in case there were other attributes which would be useful to track, but I can't think of any.  Maybe if true deprecations are added at some point, a `removedInVersion` attribute would be useful, but it's not necessary here.  I'll simplify as you suggest.